stytch/b2b/discovery/organizations/types.go

```diff
@@ -70,23 +70,23 @@ type CreateParams struct {
 	//
 	//   `NOT_ALLOWED` – disable JIT provisioning via SSO.
 	//
-	SsoJitProvisioning string `json:"sso_jit_provisioning,omitempty"`
+	SSOJITProvisioning string `json:"sso_jit_provisioning,omitempty"`
 	// EmailAllowedDomains: An array of email domains that allow invites or JIT provisioning for new Members.
 	// This list is enforced when either `email_invites` or `email_jit_provisioning` is set to `RESTRICTED`.
 	//
 	//
 	//     Common domains such as `gmail.com` are not allowed. See the
 	// [common email domains resource](https://stytch.com/docs/b2b/api/common-email-domains) for the full list.
 	EmailAllowedDomains []string `json:"email_allowed_domains,omitempty"`
-	// EmailJitProvisioning: The authentication setting that controls how a new Member can be provisioned by
+	// EmailJITProvisioning: The authentication setting that controls how a new Member can be provisioned by
```

Comment:
You're also taking the naming updates that I had planned for #132. This will get a bit trickier.

Reply:
I can wait!